test/e2e/tests/signature-request.spec.js

```diff
@@ -128,34 +218,80 @@ describe('Sign Typed Data V3 Signature Request', function () {
           windowHandles,
         );
 
-        const title = await driver.findElement(
-          '.signature-request__content__title',
+        await verifyAndAssertSignTypedDataV3(driver);
+
+        // Approve signing typed data
+        await approveSignatureRequest(
+          driver,
+          '[data-testid="signature-request-scroll-button"]',
         );
-        const origin = await driver.findElement('.signature-request__origin');
-        const verifyContractDetailsButton = await driver.findElement(
-          '.signature-request-content__verify-contract-details',
+        await driver.waitUntilXWindowHandles(2);
+        windowHandles = await driver.getAllWindowHandles();
+
+        // switch to the Dapp and verify the signed addressed
+        await driver.switchToWindowWithTitle('E2E Test Dapp', windowHandles);
+        await driver.clickElement('#signTypedDataV3Verify');
+        const recoveredAddress = await driver.findElement(
+          '#signTypedDataV3VerifyResult',
         );
+        assert.equal(await recoveredAddress.getText(), publicAddress);
+      },
+    );
+  });
+
+  it('can queue multiple Signature Requests and confirm', async function () {
```

Comment:
Why is it we refer to these typed tests more generally as `Signature Requests` rather than `typed signs`? Would that be more explicit?

Reply:
Renamed to clarify each test.